scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/core/LiveJobs.java

```diff
@@ -604,7 +605,21 @@ public TerminationData taskTerminatedWithResult(TaskId taskId, TaskResultImpl re
                         return terminationData;
                     }
                 } else if (numberOfExecutionLeft <= 0) {
-                    jobData.job.increaseNumberOfFaultyTasks(taskId);
+                    if (!onErrorPolicyInterpreter.requiresPauseTaskOnError(task) &&
+                        !onErrorPolicyInterpreter.requiresPauseJobOnError(task) &&
+                        !onErrorPolicyInterpreter.requiresCancelJobOnError(task)) {
+                        jobData.job.increaseNumberOfFaultyTasks(taskId);
```

Comment:
I have fetched the changes and performed a few manual tests. The [issue that was created by Brian](https://github.com/ow2-proactive/scheduling/issues/2782) seems to be resolved.

 However, I have noticed the _Executions_ counter can go over the maximum defined for the Task when an In-Error Task is restarted manually multiple times:

![capture d ecran de 2017-03-27 11-56-54](https://cloud.githubusercontent.com/assets/128898/24354523/71b52df4-12f2-11e7-8f28-5d07e3400939.png)
 
Is it normal? I mean an acceptable behaviour?